domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Månedsberegning.kt

```diff
@@ -12,12 +12,15 @@ interface Månedsberegning : PeriodisertInformasjon {
     fun getSatsbeløp(): Double
     fun getFradrag(): List<Fradrag>
     fun getFribeløpForEps(): Double
+    fun sumYtelseUtenSosialstønad(): Int
 
     fun erFradragForEpsBenyttetIBeregning() =
         getFradrag().any { it.fradragstype == Fradragstype.BeregnetFradragEPS }
 
     fun erSumYtelseUnderMinstebeløp() =
-        getSumYtelse() == 0 && getFradrag().any { it.fradragstype == Fradragstype.UnderMinstenivå }
+        (getSumYtelse() == 0 && getFradrag().any { it.fradragstype == Fradragstype.UnderMinstenivå }) || getSumYtelse().let {
+            it != 0 && it < Sats.toProsentAvHøy(periode)
```

Comment:
Dette vil vel føre til at beregningen som gjøres for søknadsbehandling vil føre til opphør ved revurdering for samme periode? Gitt logikken i `VurderOpphørVedRevurdering`